### DISCUSSION TITLE
Welcome to hammerspoon Discussions!

### DISCUSSION THREAD

**Discussion by cmsj** [Hammerspoon/hammerspoon#2696] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by jizampa** [comment]
Hi there, 

I have got a new Mac M1 and have copied the hammerspoon over from the old Mac.
Some of the commands work but some does not.
The one that does not work so far is:
hs.hotkey.bind(
    {"option","cmd"}, "h", function()
        hs.eventtap.keyStrokes("TEXT") end
    )
Any help?
